tcptracer-bpf.h

```diff
@@ -72,6 +72,23 @@ struct ipv6_tuple_t {
 	__u32 pid;
 };
 
+struct http_stats_t
+{
+	__u64 success_count;
+	__u64 bad_request_count;
+	__u64 server_error_count;
+};
+
+struct fd_info {
+    __u16 active;
+    __u64 start_time_ns;
+};
+
+struct log_http_request {
```

Comment:
Isn't this a `response` and not a `request`? Responses have status codes, requests don't

Reply:
gone